source/analysis/AM1DCalibrationAB.cpp

```diff
@@ -358,7 +358,7 @@ void AM1DCalibrationAB::computeCachedValues() const
 			scale = 1;
 
 		    else
-			scale = (data.at(postEdgePointValue)/normalizer.at(postEdgePointValue)) - offset;
+			scale = qAbs((data.at(postEdgePointValue)/normalizer.at(postEdgePointValue)) - offset);
```

Comment:
When the first point is higher than the last point, the spectra end up inverted.  This was a failed attempt to resolve that issue.  I didn't realize that I had left it in here.  I was just about to tackle this problem in #1158. I'll maybe have a quick look at it here.